src/engine/enginebuffer.cpp

```diff
@@ -483,6 +485,7 @@ void EngineBuffer::slotTrackLoading() {
     // Set play here, to signal the user that the play command is adopted
     m_playButton->set((double)m_bPlayAfterLoading);
     m_pTrackSamples->set(0); // Stop renderer
+    m_pTrackLoaded->set(1);
```

Comment:
It's not loaded at this point -- this signals CachingReader to load the track and see if it's valid.

We should set to zero here, then 1 in slotTrackLoaded (line 519, where we set track_samples).